<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Ahmads-Laptop" failures="0" tests="2" name="test.UserAuthenticationTests" time="1.772" errors="1" timestamp="2024-11-02T23:39:12 PKT" skipped="0">
  <testcase classname="test.UserAuthenticationTests" name="testLoginWithExcelData" time="0.046"/>
  <system-out/>
  <testcase classname="test.UserAuthenticationTests" name="testLoginWithExcelData" time="1.726">
    <error message="
[public void test.UserAuthenticationTests.testLoginWithExcelData(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,boolean)] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: testLoginWithExcelData([Parameter{index=0, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=1, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=2, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=3, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=4, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=5, type=boolean, declaredAnnotations=[]}])
Arguments: [(java.lang.Double) 1.0,(java.lang.String) login,(java.lang.String) valid login,(java.lang.String) user,(java.lang.String) pass,(java.lang.Boolean) true]" type="org.testng.internal.reflect.MethodMatcherException">
      <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
[public void test.UserAuthenticationTests.testLoginWithExcelData(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,boolean)] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: testLoginWithExcelData([Parameter{index=0, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=1, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=2, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=3, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=4, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=5, type=boolean, declaredAnnotations=[]}])
Arguments: [(java.lang.Double) 1.0,(java.lang.String) login,(java.lang.String) valid login,(java.lang.String) user,(java.lang.String) pass,(java.lang.Boolean) true]
at org.testng.internal.reflect.DataProviderMethodMatcher.getConformingArguments(DataProviderMethodMatcher.java:43)
at org.testng.internal.Parameters.injectParameters(Parameters.java:914)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:34)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1570)
]]>
    </error>
  </testcase> <!-- testLoginWithExcelData -->
  <system-out/>
</testsuite> <!-- test.UserAuthenticationTests -->
